packages/user-ui/src/pages/projects/_id/discussion/_id/page.rs

```diff
@@ -47,19 +49,20 @@ pub fn DiscussionVideoPage(
                     }
                 }
 
-                //FIXME: fix to real email
                 ParticipantSidebar {
                     show_member: show_side_member(),
                     hide_member: move |_| {
                         show_side_member.set(false);
                     },
-                    emails: vec![
-                        "aaa@bbb.ccc".to_string(),
-                        "aaa1@bbb.ccc".to_string(),
-                        "aaa2@bbb.ccc".to_string(),
-                        "aaa3@bbb.ccc".to_string(),
-                        "aaa4@bbb.ccc".to_string(),
-                    ],
+                    refresh: move |_| {
+                        ctrl.refresh();
+                    },
```

Comment:
```suggestion
                    onrefresh: move |_| {
                        ctrl.handle_refresh();
                    },
```

보통 이벤트 기반의 함수를 사용할때는 같이 어울리는 키워드들이 있습니다.

- emit: 이벤트를 발생 시킬때 사용
- on: 특정 이벤트가 발생했을때 사용
- handle: 특정 이벤트에 대한 로직을 처리할 때 사용

이런 관점을 보시면 좋을거 같고, event 기반 개발에서 주로 어떤 용어들이 사용되는지 한번 공부해보시면 좋을듯 합니다.

Reply:
넵 수정했습니다!